### ISSUE TITLE
Deleted mail from inbox is doubled in trash directory

### ISSUE THREAD

**Issue by gandogar** [thunderbird/thunderbird-android#2705]
Please search to check for an existing issue (including closed issues, for which the fix may not have yet been released) before opening a new issue: https://github.com/k9mail/k-9/issues?q=is%3Aissue

### Expected behavior
When deleting one or more mails from inbox, I expect every deleted mail one time in trash directory.

### Actual behavior
When deleting one or more mails from inbox, every deleted mail exist in trash two times.

### Steps to reproduce
1. Wait for incomaing mail  (or send a test mail)
2. Delete a mail from inbox using symbol 'trash can' on left bottom.
3. Change to 'trash' directory. 
    In first moment the deleted mail appears only one time. After short time the deleted mail appears a
    second time in trash directory.
    After manually syncing the doubled mail disappears - the mail is only shown one time.

### Environment
K-9 Mail version: 5.207
I use the same email account with thunderbird on my desktop pc. These problems do not occur
using thunderbird on pc.

Android version: 7.0.0

Account type (IMAP, POP3, WebDAV/Exchange):
It is an IMAP Account.

**Comment by cketti** [comment]
Can you please record a debug log when deleting a message. See https://github.com/k9mail/k-9/wiki/LoggingErrors

**Comment by gandogar** [comment]
Hello,

I tried to attach the log file. But there was a message:
"Something went really wrong, and we can't process that file".

What can I do?
The file is named 'k9-log.txt', it is about 623 kb.
Is that too much?

gandogar

**Comment by gandogar** [comment]
I got it. Problem was a script-blocker in my browser.

[k9-log.zip](https://github.com/k9mail/k-9/files/1256591/k9-log.zip)

gandogar

**Comment by philipwhiuk** [comment]
```
08-28 12:08:17.733 I/WindowManager( 1043): Destroying surface Surface(name=com.fsck.k9/com.fsck.k9.activity.Accounts) called by com.android.server.wm.WindowStateAnimator.destroySurface:2135 com.android.server.wm.WindowStateAnimator.destroySurfaceLocked:947 com.android.server.wm.WindowState.removeLocked:1461 com.android.server.wm.WindowManagerService.removeWindowInnerLocked:2715 com.android.server.wm.WindowManagerService.removeWindowLocked:2663 com.android.server.wm.WindowManagerService.removeWindowLocked:2529 com.android.server.wm.WindowManagerService.removeWindow:2524 com.android.server.wm.Session.remove:196 
08-28 12:08:17.750 I/ActivityManager( 1043): Killing 2998:com.fsck.k9/u0a175 (adj 1001): remove task
08-28 12:08:17.885 D/ActivityManager( 1043): handleAppDiedLocked: app = ProcessRecord{3259cd 2998:com.fsck.k9/u0a175}, app.pid = 2998
08-28 12:08:17.885 W/ActivityManager( 1043): Scheduling restart of crashed service com.fsck.k9/.service.PushService in 1000ms
08-28 12:08:18.901 I/ActivityManager( 1043): Start proc 5819:com.fsck.k9/u0a175 for service com.fsck.k9/.service.PushService
08-28 12:08:19.026 V/ActivityThread( 5819): SVC-Creating service CreateServiceData{token=android.os.BinderProxy@789c384 className=com.fsck.k9.service.PushService packageName=com.fsck.k9 intent=null}
08-28 12:08:19.027 D/ActivityThread( 5819): SVC-Calling onStartCommand: com.fsck.k9.service.PushService@b52a76d, flags=0, startId=4
08-28 12:08:19.028 V/ActivityThread( 5819): SVC-Creating service CreateServiceData{token=android.os.BinderProxy@c82afa2 className=com.fsck.k9.service.MailService packageName=com.fsck.k9 intent=null}
08-28 12:08:19.029 D/ActivityThread( 5819): SVC-Calling onStartCommand: com.fsck.k9.service.MailService@e745433, flags=0, startId=1
08-28 12:08:19.038 V/ActivityThread( 5819): SVC-Destroying service com.fsck.k9.service.PushService@b52a76d
08-28 12:08:19.052 V/ActivityManager( 1043): Broadcast: Intent { act=com.fsck.k9.service.BroadcastReceiver.scheduleIntent flg=0x10 cmp=com.fsck.k9/.service.BootReceiver (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:19.055 D/ActivityThread( 5819): BDC-Calling onReceive: intent=Intent { act=com.fsck.k9.service.BroadcastReceiver.scheduleIntent flg=0x10 cmp=com.fsck.k9/.service.BootReceiver (has extras) }, receiver=com.fsck.k9.service.BootReceiver@dce90fa
08-28 12:08:19.062 V/AlarmManager( 1043): APP set(PendingIntent{2127345: PendingIntentRecord{27f73bd com.fsck.k9 broadcastIntent}}) : type=0 triggerAtTime=1503921304140 win=-1 tElapsed=25497848 maxElapsed=30301656 interval=0 flags=0x8
08-28 12:08:19.062 D/AlarmManager( 1043): remove(operation) changed bounds; rebatching operation = PendingIntent{2127345: PendingIntentRecord{27f73bd com.fsck.k9 broadcastIntent}}
08-28 12:08:19.065 V/ActivityThread( 5819): SVC-Creating service CreateServiceData{token=android.os.BinderProxy@2fff108 className=com.fsck.k9.service.PushService packageName=com.fsck.k9 intent=null}
08-28 12:08:19.065 D/ActivityThread( 5819): SVC-Calling onStartCommand: com.fsck.k9.service.PushService@69a40a1, flags=0, startId=1
08-28 12:08:19.070 V/ActivityThread( 5819): SVC-Destroying service com.fsck.k9.service.PushService@69a40a1
08-28 12:08:19.128 V/ActivityThread( 5819): SVC-Creating service CreateServiceData{token=android.os.BinderProxy@f4962d9 className=com.fsck.k9.service.PushService packageName=com.fsck.k9 intent=null}
08-28 12:08:19.129 D/ActivityThread( 5819): SVC-Calling onStartCommand: com.fsck.k9.service.PushService@b33649e, flags=0, startId=1
08-28 12:08:19.130 V/ActivityManager( 1043): Broadcast: Intent { act=com.fsck.k9.service.BroadcastReceiver.scheduleIntent flg=0x10 cmp=com.fsck.k9/.service.BootReceiver (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:19.139 D/ActivityThread( 5819): BDC-Calling onReceive: intent=Intent { act=com.fsck.k9.service.BroadcastReceiver.scheduleIntent flg=0x10 cmp=com.fsck.k9/.service.BootReceiver (has extras) }, receiver=com.fsck.k9.service.BootReceiver@dbe2b7f
08-28 12:08:19.143 V/AlarmManager( 1043): APP set(PendingIntent{26edfcb: PendingIntentRecord{72b1784 com.fsck.k9 broadcastIntent}}) : type=0 triggerAtTime=1503916339128 win=-1 tElapsed=20532837 maxElapsed=21612825 interval=0 flags=0x8
08-28 12:08:19.144 D/AlarmManager( 1043): remove(operation) changed bounds; rebatching operation = PendingIntent{26edfcb: PendingIntentRecord{72b1784 com.fsck.k9 broadcastIntent}}
08-28 12:08:19.148 V/ActivityThread( 5819): SVC-Destroying service com.fsck.k9.service.MailService@e745433
08-28 12:08:19.725 V/ActivityManager( 1043): Broadcast: Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:19.727 V/ActivityManager( 1043): Broadcast: Intent { act=com.fsck.k9.intent.action.REFRESH_OBSERVER flg=0x10 (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:19.727 D/ActivityThread( 5819): BDC-Calling onReceive: intent=Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) }, receiver=com.fsck.k9.provider.UnreadWidgetProvider@c295b73
08-28 12:08:19.815 V/ActivityManager( 1043): Broadcast: Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:19.817 V/ActivityManager( 1043): Broadcast: Intent { act=com.fsck.k9.intent.action.REFRESH_OBSERVER flg=0x10 (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:19.818 D/ActivityThread( 5819): BDC-Calling onReceive: intent=Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) }, receiver=com.fsck.k9.provider.UnreadWidgetProvider@24a1e1
08-28 12:08:20.060 V/ActivityManager( 1043): Broadcast: Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:20.062 V/ActivityManager( 1043): Broadcast: Intent { act=com.fsck.k9.intent.action.REFRESH_OBSERVER flg=0x10 (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:20.063 D/ActivityThread( 5819): BDC-Calling onReceive: intent=Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) }, receiver=com.fsck.k9.provider.UnreadWidgetProvider@ea5cbfd
08-28 12:08:20.152 V/ActivityManager( 1043): Broadcast: Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:20.156 D/ActivityThread( 5819): BDC-Calling onReceive: intent=Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) }, receiver=com.fsck.k9.provider.UnreadWidgetProvider@d1f4d8
08-28 12:08:20.157 V/ActivityManager( 1043): Broadcast: Intent { act=com.fsck.k9.intent.action.REFRESH_OBSERVER flg=0x10 (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:20.201 V/ActivityManager( 1043): Broadcast: Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:20.205 V/ActivityManager( 1043): Broadcast: Intent { act=com.fsck.k9.intent.action.REFRESH_OBSERVER flg=0x10 (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:20.207 D/ActivityThread( 5819): BDC-Calling onReceive: intent=Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) }, receiver=com.fsck.k9.provider.UnreadWidgetProvider@8bdceee
08-28 12:08:20.391 V/ActivityManager( 1043): Broadcast: Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:20.394 V/ActivityManager( 1043): Broadcast: Intent { act=com.fsck.k9.intent.action.REFRESH_OBSERVER flg=0x10 (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:20.394 D/ActivityThread( 5819): BDC-Calling onReceive: intent=Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) }, receiver=com.fsck.k9.provider.UnreadWidgetProvider@1d0cdd
08-28 12:08:20.482 V/ActivityManager( 1043): Broadcast: Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:20.488 D/ActivityThread( 5819): BDC-Calling onReceive: intent=Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) }, receiver=com.fsck.k9.provider.UnreadWidgetProvider@be8bf38
08-28 12:08:20.491 V/ActivityManager( 1043): Broadcast: Intent { act=com.fsck.k9.intent.action.REFRESH_OBSERVER flg=0x10 (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:20.577 V/ActivityManager( 1043): Broadcast: Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:20.580 V/ActivityManager( 1043): Broadcast: Intent { act=com.fsck.k9.intent.action.REFRESH_OBSERVER flg=0x10 (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:08:20.580 D/ActivityThread( 5819): BDC-Calling onReceive: intent=Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) }, receiver=com.fsck.k9.provider.UnreadWidgetProvider@8bf5c4e
08-28 12:08:34.129 I/ActivityManager( 1043): Waited long enough for: ServiceRecord{d584a9a u0 com.fsck.k9/.service.PushService}
08-28 12:10:50.903 V/ActivityManager( 1043): Broadcast: Intent { act=com.fsck.k9.intent.action.EMAIL_RECEIVED dat=email://messages/0/INBOX/11211 flg=0x10 (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:10:50.905 V/ActivityManager( 1043): Broadcast: Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:10:50.909 D/ActivityThread( 5819): BDC-Calling onReceive: intent=Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) }, receiver=com.fsck.k9.provider.UnreadWidgetProvider@8706354
08-28 12:10:50.968 V/NotificationService( 1043): enqueueNotificationInternal: pkg=com.fsck.k9 id=7 notification=Notification(pri=0 contentView=null vibrate=null sound=null tick defaults=0x0 flags=0x10 color=0xff00ff43 category=email groupKey=newMailNotifications-0 actions=3 vis=PRIVATE)
08-28 12:10:50.969 D/NotificationService( 1043): EnqueueNotificationRunnable.run for: 0|com.fsck.k9|7|null|10175
08-28 12:10:50.970 V/mmm     ( 1043): EnqueueNotificationRunnable, run():, pkg=com.fsck.k9, time=2017-08-28(Mo.) 12:10:50
08-28 12:10:50.972 V/NotificationService( 1043): pkg=com.fsck.k9 canInterrupt=false intercept=false
08-28 12:10:51.010 V/NotificationService( 1043): enqueueNotificationInternal: pkg=com.fsck.k9 id=6 notification=Notification(pri=0 contentView=null vibrate=null sound=content://settings/system/notification_sound tick defaults=0x0 flags=0x210 color=0xff00ff43 category=email groupKey=newMailNotifications-0 actions=3 vis=PRIVATE publicVersion=Notification(pri=0 contentView=com.fsck.k9/0x109008b vibrate=null sound=null defaults=0x0 flags=0x0 color=0xff00ff43 category=email vis=PRIVATE))
08-28 12:10:51.011 D/NotificationService( 1043): EnqueueNotificationRunnable.run for: 0|com.fsck.k9|6|null|10175
08-28 12:10:51.012 V/mmm     ( 1043): EnqueueNotificationRunnable, run():, pkg=com.fsck.k9, time=2017-08-28(Mo.) 12:10:51
08-28 12:10:51.015 V/NotificationService( 1043): pkg=com.fsck.k9 canInterrupt=false intercept=false
08-28 12:10:51.086 V/ActivityManager( 1043): Broadcast: Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:10:51.092 D/ActivityThread( 5819): BDC-Calling onReceive: intent=Intent { act=android.appwidget.action.APPWIDGET_UPDATE flg=0x10 cmp=com.fsck.k9/.provider.UnreadWidgetProvider (has extras) }, receiver=com.fsck.k9.provider.UnreadWidgetProvider@a4f2d4c
08-28 12:10:51.107 V/ActivityManager( 1043): Broadcast: Intent { act=com.fsck.k9.intent.action.REFRESH_OBSERVER flg=0x10 (has extras) } ordered=false userid=0 callerApp=ProcessRecord{80877af 5819:com.fsck.k9/u0a175}
08-28 12:12:51.503 V/k9      ( 5819): conn109715847<<<#null# [OK, Still here]
08-28 12:12:51.503 V/k9      ( 5819): Got async response: #null# [OK, Still here]
08-28 12:12:51.503 V/k9      ( 5819): conn172004746<<<#null# [OK, Still here]
08-28 12:12:51.503 V/k9      ( 5819): Got async response: #null# [OK, Still here]
08-28 12:12:51.504 V/k9      ( 5819): conn74553524<<<#null# [OK, Still here]
08-28 12:12:51.504 V/k9      ( 5819): Got async response: #null# [OK, Still here]
08-28 12:12:51.532 V/k9      ( 5819): conn123555619<<<#null# [OK, Still here]
08-28 12:12:51.532 V/k9      ( 5819): Got async response: #null# [OK, Still here]
08-28 12:12:51.874 V/k9      ( 5819): conn71217097<<<#null# [OK, Still here]
08-28 12:12:51.874 V/k9      ( 5819): Got async response: #null# [OK, Still here]
08-28 12:12:52.075 V/k9      ( 5819): conn95511362<<<#null# [OK, Still here]
08-28 12:12:52.075 V/k9      ( 5819): Got async response: #null# [OK, Still here]
08-28 12:13:45.005 I/        ( 5951): service_to_fd shell:export ANDROID_LOG_TAGS="''"; exec logcat '-d' '-v' 'time' 'k9:V' ''\'''\'*:S'' 'AndroidRuntime:E'
```

**Comment by philipwhiuk** [comment]
There's no stacktrace. Are you running a custom ROM?

**Comment by gandogar** [comment]
No. I am running the original stock rom which was delivered with Nokia 3.
I deactivated many (google-)apps, which I do not need.

**Comment by cketti** [comment]
The log doesn't contain any useful information. Did you record it while deleting a message and then refreshing the Trash folder?

**Comment by gandogar** [comment]
How to start the recording? I activated in K-9 global settings "debug-protocoll" ("Fehlersuchprotokoll").
Must I set further settings? 

I did it again as follows:
I opened K-9 Mail at 16:25 pm (local time, see timestamps in logfile).
In the inbox I deleted a mail, changed into trash directory and waited a few seconds.
Then the deleted mail appeared twice.
After a manual sync the mail appeared only once again.
All this happend between 16:25:00 pm and 16:26:00 pm. So I shortened the logfile.

gandogar

[k9-log_1625.zip](https://github.com/k9mail/k-9/files/1257125/k9-log_1625.zip)

**Comment by rancidfrog** [comment]
This also happens to sent mail and drafts.
There is an issue with sync I believe.
I am able to replicate by:

1. Create a draft and Save Draft
![](https://s26.postimg.org/nox7wvrux/Screenshot_20170908-182822.png)
2. Refresh Draft folder
3. Delete draft, (You will see a copy of draft in Trash folder)
4. Refresh Trash folder, (Another copy of same draft appears)
![](https://s26.postimg.org/r797t9sqx/Screenshot_20170908-182902.png)
There are three Hello mails in image above. That is because I did not refresh/sync draft folder before deleting first Hello draft. So I repeated process with refresh/sync in between, which is why I believe it is an issue with syncing
```
09-08 18:28:47.499 V/k9      (25507): conn111226805<<<#null# [290, FETCH, [UID, 765, FLAGS, [\Recent], INTERNALDATE, 08-Sep-2017 12:26:56 -0300, RFC822.SIZE, 3401, BODY, [HEADER.FIELDS, [date, subject, from, content-type, to, cc, reply-to, message-id, references, in-reply-to, X-K9mail-Identity]], Date: Fri, 08 Sep 2017 15:27:15 +0000
09-08 18:28:47.499 V/k9      (25507): From: "xxx" <xxx>
09-08 18:28:47.499 V/k9      (25507): Subject: Hello
09-08 18:28:47.499 V/k9      (25507): Message-ID: <920E71AC-76F5-4E35-A6D8-24FED9D1EC6C@xxx.com>
09-08 18:28:47.499 V/k9      (25507): Content-Type: multipart/alternative; boundary="----H9BJWZFRZ3XF48PYD4BG8N0Y2UEBGG"
09-08 18:28:47.499 V/k9      (25507): X-K9mail-Identity: !l=0&o=0&pl=0&po=0&qs=PREFIX&f=HTML&p=0&q=NONE
09-08 18:28:47.499 V/k9      (25507): 
09-08 18:28:47.499 V/k9      (25507): ]]
09-08 18:28:47.499 V/k9      (25507): Stored uid '765' for msgSeq 290 into map
09-08 18:28:47.500 V/k9      (25507): conn111226805<<<#null# [291, FETCH, [UID, 766, FLAGS, [\Recent], INTERNALDATE, 08-Sep-2017 12:27:42 -0300, RFC822.SIZE, 3419, BODY, [HEADER.FIELDS, [date, subject, from, content-type, to, cc, reply-to, message-id, references, in-reply-to, X-K9mail-Identity]], Date: Fri, 08 Sep 2017 15:28:10 +0000
09-08 18:28:47.500 V/k9      (25507): From: "xxx" <xxx>
09-08 18:28:47.500 V/k9      (25507): Subject: Hello 2
09-08 18:28:47.500 V/k9      (25507): Message-ID: <ECA708D1-9985-4678-A0CD-A04A46528956@xxx.com>
09-08 18:28:47.500 V/k9      (25507): Content-Type: multipart/alternative; boundary="----SWHJI1304LP3AUB3MQC7M6VETN5CMT"
09-08 18:28:47.500 V/k9      (25507): X-K9mail-Identity: !l=0&o=0&pl=0&po=0&qs=PREFIX&f=HTML&p=0&q=NONE
09-08 18:28:47.500 V/k9      (25507): 
09-08 18:28:47.500 V/k9      (25507): ]]
09-08 18:28:47.500 V/k9      (25507): Stored uid '766' for msgSeq 291 into map
09-08 18:28:47.693 V/k9      (25507): conn111226805<<<#57# [OK, FETCH completed.]
09-08 18:28:47.705 D/k9      (25507): SYNC: Synced unsynced messages for folder Trash
09-08 18:28:47.705 D/k9      (25507): SYNC: Have 10 large messages and 3 small messages out of 13 unsynced messages
09-08 18:28:47.705 D/k9      (25507): SYNC: Fetching 3 small messages for folder Trash
09-08 18:28:47.706 V/k9      (25507): conn111226805>>> 58 UID FETCH 747,765,766 (UID BODY.PEEK[])
09-08 18:28:48.079 V/k9      (25507): conn111226805<<<#null# [272, FETCH, [UID, 747, BODY, [], 1]]
09-08 18:28:48.080 V/k9      (25507): Stored uid '747' for msgSeq 272 into map
09-08 18:28:48.105 D/k9      (25507): deleting temp file
09-08 18:28:48.108 D/k9      (25507): deleting temp file
09-08 18:28:48.133 V/k9      (25507): LockableDatabase: Transaction ended, took 13ms / com.fsck.k9.mailstore.LocalFolder.storeSmallMessage(LocalFolder.java:1151)
09-08 18:28:48.133 V/k9      (25507): About to notify listeners that we got a new small message Mo"D:Trash:747
09-08 18:28:48.136 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_RECEIVED account=Mo"D folder=Trash message uid=747
09-08 18:28:48.141 V/k9      (25507): conn111226805<<<#null# [290, FETCH, [UID, 765, BODY, [], 1]]
09-08 18:28:48.141 V/k9      (25507): Stored uid '765' for msgSeq 290 into map
09-08 18:28:48.154 D/k9      (25507): deleting temp file
09-08 18:28:48.156 D/k9      (25507): deleting temp file
09-08 18:28:48.180 V/k9      (25507): LockableDatabase: Transaction ended, took 14ms / com.fsck.k9.mailstore.LocalFolder.storeSmallMessage(LocalFolder.java:1151)
09-08 18:28:48.180 V/k9      (25507): About to notify listeners that we got a new small message Mo"D:Trash:765
09-08 18:28:48.181 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_RECEIVED account=Mo"D folder=Trash message uid=765
09-08 18:28:48.185 V/k9      (25507): conn111226805<<<#null# [291, FETCH, [UID, 766, BODY, [], 1]]
09-08 18:28:48.185 V/k9      (25507): Stored uid '766' for msgSeq 291 into map
09-08 18:28:48.192 D/k9      (25507): deleting temp file
09-08 18:28:48.193 D/k9      (25507): deleting temp file
09-08 18:28:48.211 V/k9      (25507): LockableDatabase: Transaction ended, took 11ms / com.fsck.k9.mailstore.LocalFolder.storeSmallMessage(LocalFolder.java:1151)
09-08 18:28:48.211 V/k9      (25507): About to notify listeners that we got a new small message Mo"D:Trash:766
09-08 18:28:48.213 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_RECEIVED account=Mo"D folder=Trash message uid=766
09-08 18:28:48.267 V/k9      (25507): conn111226805<<<#58# [OK, FETCH completed.]
09-08 18:28:48.267 D/k9      (25507): SYNC: Done fetching small messages for folder Trash
09-08 18:28:48.267 D/k9      (25507): SYNC: Fetching large messages for folder Trash
09-08 18:28:48.268 V/k9      (25507): conn111226805>>> 59 UID FETCH 736,737,738,739,740,753,754,755,756,760 (UID BODYSTRUCTURE)
09-08 18:28:48.677 V/k9      (25507): conn111226805<<<#null# [267, FETCH, [UID, 736, BODYSTRUCTURE, [[text, plain, [charset, ISO-8859-1], NIL, NIL, 8bit, 245, 12, NIL, NIL, NIL, NIL], [text, html, [charset, ISO-8859-1], NIL, NIL, 8bit, 2642, 34, NIL, NIL, NIL, NIL], alternative, [boundary, b1_f20f6ea97589f59842fef7dda32339f0], NIL, NIL]]]
09-08 18:28:48.677 V/k9      (25507): Stored uid '736' for msgSeq 267 into map
09-08 18:28:48.679 V/k9      (25507): conn111226805<<<#null# [268, FETCH, [UID, 737, BODYSTRUCTURE, [text, html, [charset, UTF-8], NIL, NIL, 7bit, 1312, 34, NIL, NIL, NIL, NIL]]]
09-08 18:28:48.679 V/k9      (25507): Stored uid '737' for msgSeq 268 into map
09-08 18:28:48.680 V/k9      (25507): conn111226805<<<#null# [269, FETCH, [UID, 738, BODYSTRUCTURE, [text, plain, [charset, iso-8859-1], NIL, NIL, 8bit, 64, 2, NIL, NIL, NIL, NIL]]]
09-08 18:28:48.680 V/k9      (25507): Stored uid '738' for msgSeq 269 into map
09-08 18:28:48.682 V/k9      (25507): conn111226805<<<#null# [270, FETCH, [UID, 739, BODYSTRUCTURE, [[text, plain, NIL, NIL, NIL, quoted-printable, 1539, 22, NIL, NIL, NIL, NIL], [text, html, NIL, NIL, NIL, quoted-printable, 20615, 302, NIL, NIL, NIL, NIL], alternative, [boundary, --boundary_1652728_769321f0-ac56-407b-87e3-a4053a3367f5], NIL, NIL]]]
09-08 18:28:48.682 V/k9      (25507): Stored uid '739' for msgSeq 270 into map
09-08 18:28:48.684 V/k9      (25507): conn111226805<<<#null# [271, FETCH, [UID, 740, BODYSTRUCTURE, [[text, plain, NIL, NIL, NIL, quoted-printable, 464, 6, NIL, NIL, NIL, NIL], [text, html, NIL, NIL, NIL, quoted-printable, 7815, 114, NIL, NIL, NIL, NIL], alternative, [boundary, --boundary_1989439_2af1f696-932b-4d48-b07b-2a5002930a3d], NIL, NIL]]]
09-08 18:28:48.684 V/k9      (25507): Stored uid '740' for msgSeq 271 into map
09-08 18:28:48.686 V/k9      (25507): conn111226805<<<#null# [278, FETCH, [UID, 753, BODYSTRUCTURE, [[text, plain, [charset, UTF-8, format, flowed, delsp, yes], NIL, NIL, base64, 2130, 27, NIL, NIL, NIL, NIL], [text, html, [charset, UTF-8], NIL, NIL, quoted-printable, 6136, 80, NIL, NIL, NIL, NIL], alternative, [boundary, 001a1142a202f476130556386203], NIL, NIL]]]
09-08 18:28:48.686 V/k9      (25507): Stored uid '753' for msgSeq 278 into map
09-08 18:28:48.688 V/k9      (25507): conn111226805<<<#null# [279, FETCH, [UID, 754, BODYSTRUCTURE, [[text, plain, [charset, UTF-8, format, flowed, delsp, yes], NIL, NIL, base64, 2716, 34, NIL, NIL, NIL, NIL], [text, html, [charset, UTF-8], NIL, NIL, quoted-printable, 8240, 107, NIL, NIL, NIL, NIL], alternative, [boundary, 001a114dc662c6b18205564254c4], NIL, NIL]]]
09-08 18:28:48.688 V/k9      (25507): Stored uid '754' for msgSeq 279 into map
09-08 18:28:48.691 V/k9      (25507): conn111226805<<<#null# [280, FETCH, [UID, 755, BODYSTRUCTURE, [[text, plain, [charset, UTF-8, format, flowed, delsp, yes], NIL, NIL, base64, 2104, 26, NIL, NIL, NIL, NIL], [text, html, [charset, UTF-8], NIL, NIL, quoted-printable, 6116, 80, NIL, NIL, NIL, NIL], alternative, [boundary, 001a114374d6283c2a0556425940], NIL, NIL]]]
09-08 18:28:48.691 V/k9      (25507): Stored uid '755' for msgSeq 280 into map
09-08 18:28:48.693 V/k9      (25507): conn111226805<<<#null# [281, FETCH, [UID, 756, BODYSTRUCTURE, [[text, plain, [charset, UTF-8, format, flowed, delsp, yes], NIL, NIL, base64, 2104, 26, NIL, NIL, NIL, NIL], [text, html, [charset, UTF-8], NIL, NIL, quoted-printable, 6116, 80, NIL, NIL, NIL, NIL], alternative, [boundary, 001a1142527e3934f50556425eac], NIL, NIL]]]
09-08 18:28:48.694 V/k9      (25507): Stored uid '756' for msgSeq 281 into map
09-08 18:28:48.696 V/k9      (25507): conn111226805<<<#null# [285, FETCH, [UID, 760, BODYSTRUCTURE, [[text, plain, [charset, UTF-8, format, flowed, delsp, yes], NIL, NIL, base64, 2104, 26, NIL, NIL, NIL, NIL], [text, html, [charset, UTF-8], NIL, NIL, quoted-printable, 6116, 80, NIL, NIL, NIL, NIL], alternative, [boundary, 001a114dce20f893b70556426df9], NIL, NIL]]]
09-08 18:28:48.696 V/k9      (25507): Stored uid '760' for msgSeq 285 into map
09-08 18:28:48.900 V/k9      (25507): conn111226805<<<#59# [OK, FETCH completed.]
09-08 18:28:48.911 V/k9      (25507): conn111226805>>> 60 UID FETCH 736 (UID BODY.PEEK[1])
09-08 18:28:49.278 V/k9      (25507): conn111226805<<<#null# [267, FETCH, [UID, 736, BODY, [1], com.fsck.k9.mail.internet.BinaryTempFileBody@f58825f]]
09-08 18:28:49.534 V/k9      (25507): conn111226805<<<#60# [OK, FETCH completed.]
09-08 18:28:49.535 V/k9      (25507): conn111226805>>> 61 UID FETCH 736 (UID BODY.PEEK[2])
09-08 18:28:49.898 V/k9      (25507): conn111226805<<<#null# [267, FETCH, [UID, 736, BODY, [2], com.fsck.k9.mail.internet.BinaryTempFileBody@4e2a4ac]]
09-08 18:28:50.094 V/k9      (25507): conn111226805<<<#61# [OK, FETCH completed.]
09-08 18:28:50.136 D/k9      (25507): deleting temp file
09-08 18:28:50.139 D/k9      (25507): deleting temp file
09-08 18:28:50.160 V/k9      (25507): LockableDatabase: Transaction ended, took 15ms / com.fsck.k9.mailstore.LocalFolder.appendMessages(LocalFolder.java:1255)
09-08 18:28:50.169 V/k9      (25507): LockableDatabase: Transaction ended, took 5ms / com.fsck.k9.mailstore.LocalMessage.setFlag(LocalMessage.java:261)
09-08 18:28:50.171 V/k9      (25507): About to notify listeners that we got a new large message Mo"D:Trash:736
09-08 18:28:50.176 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_RECEIVED account=Mo"D folder=Trash message uid=736
09-08 18:28:50.180 V/k9      (25507): conn111226805>>> 62 UID FETCH 737 (UID BODY.PEEK[TEXT]<0.32768>)
09-08 18:28:50.569 V/k9      (25507): conn111226805<<<#null# [268, FETCH, [UID, 737, BODY, [TEXT], <0>, com.fsck.k9.mail.internet.BinaryTempFileBody@8b9190a]]
09-08 18:28:50.773 V/k9      (25507): conn111226805<<<#62# [OK, FETCH completed.]
09-08 18:28:50.816 D/k9      (25507): deleting temp file
09-08 18:28:50.836 V/k9      (25507): LockableDatabase: Transaction ended, took 12ms / com.fsck.k9.mailstore.LocalFolder.appendMessages(LocalFolder.java:1255)
09-08 18:28:50.846 V/k9      (25507): LockableDatabase: Transaction ended, took 5ms / com.fsck.k9.mailstore.LocalMessage.setFlag(LocalMessage.java:261)
09-08 18:28:50.847 V/k9      (25507): About to notify listeners that we got a new large message Mo"D:Trash:737
09-08 18:28:50.852 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_RECEIVED account=Mo"D folder=Trash message uid=737
09-08 18:28:50.858 V/k9      (25507): conn111226805>>> 63 UID FETCH 738 (UID BODY.PEEK[TEXT]<0.32768>)
09-08 18:28:51.259 V/k9      (25507): conn111226805<<<#null# [269, FETCH, [UID, 738, BODY, [TEXT], <0>, com.fsck.k9.mail.internet.BinaryTempFileBody@6ca6fae]]
09-08 18:28:51.454 V/k9      (25507): conn111226805<<<#63# [OK, FETCH completed.]
09-08 18:28:51.491 D/k9      (25507): deleting temp file
09-08 18:28:51.513 V/k9      (25507): LockableDatabase: Transaction ended, took 16ms / com.fsck.k9.mailstore.LocalFolder.appendMessages(LocalFolder.java:1255)
09-08 18:28:51.523 V/k9      (25507): LockableDatabase: Transaction ended, took 6ms / com.fsck.k9.mailstore.LocalMessage.setFlag(LocalMessage.java:261)
09-08 18:28:51.525 V/k9      (25507): About to notify listeners that we got a new large message Mo"D:Trash:738
09-08 18:28:51.528 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_RECEIVED account=Mo"D folder=Trash message uid=738
09-08 18:28:51.533 V/k9      (25507): conn111226805>>> 64 UID FETCH 739 (UID BODY.PEEK[1])
09-08 18:28:51.907 V/k9      (25507): conn111226805<<<#null# [270, FETCH, [UID, 739, BODY, [1], com.fsck.k9.mail.internet.BinaryTempFileBody@13a1f74]]
09-08 18:28:52.143 V/k9      (25507): conn111226805<<<#64# [OK, FETCH completed.]
09-08 18:28:52.144 V/k9      (25507): conn111226805>>> 65 UID FETCH 739 (UID BODY.PEEK[2])
09-08 18:28:52.526 V/k9      (25507): conn111226805<<<#null# [270, FETCH, [UID, 739, BODY, [2], com.fsck.k9.mail.internet.BinaryTempFileBody@30cec9d]]
09-08 18:28:52.527 V/k9      (25507): conn111226805<<<#65# [OK, FETCH completed.]
09-08 18:28:52.563 D/k9      (25507): deleting temp file
09-08 18:28:52.589 V/k9      (25507): LockableDatabase: Transaction ended, took 15ms / com.fsck.k9.mailstore.LocalFolder.appendMessages(LocalFolder.java:1255)
09-08 18:28:52.601 V/k9      (25507): LockableDatabase: Transaction ended, took 7ms / com.fsck.k9.mailstore.LocalMessage.setFlag(LocalMessage.java:261)
09-08 18:28:52.603 V/k9      (25507): About to notify listeners that we got a new large message Mo"D:Trash:739
09-08 18:28:52.608 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_RECEIVED account=Mo"D folder=Trash message uid=739
09-08 18:28:52.615 V/k9      (25507): conn111226805>>> 66 UID FETCH 740 (UID BODY.PEEK[1])
09-08 18:28:53.018 V/k9      (25507): conn111226805<<<#null# [271, FETCH, [UID, 740, BODY, [1], com.fsck.k9.mail.internet.BinaryTempFileBody@550b599]]
09-08 18:28:53.217 V/k9      (25507): conn111226805<<<#66# [OK, FETCH completed.]
09-08 18:28:53.218 V/k9      (25507): conn111226805>>> 67 UID FETCH 740 (UID BODY.PEEK[2])
09-08 18:28:53.580 V/k9      (25507): conn111226805<<<#null# [271, FETCH, [UID, 740, BODY, [2], com.fsck.k9.mail.internet.BinaryTempFileBody@808205e]]
09-08 18:28:53.776 V/k9      (25507): conn111226805<<<#67# [OK, FETCH completed.]
09-08 18:28:53.817 D/k9      (25507): deleting temp file
09-08 18:28:53.820 D/k9      (25507): deleting temp file
09-08 18:28:53.845 V/k9      (25507): LockableDatabase: Transaction ended, took 17ms / com.fsck.k9.mailstore.LocalFolder.appendMessages(LocalFolder.java:1255)
09-08 18:28:53.860 V/k9      (25507): LockableDatabase: Transaction ended, took 10ms / com.fsck.k9.mailstore.LocalMessage.setFlag(LocalMessage.java:261)
09-08 18:28:53.861 V/k9      (25507): About to notify listeners that we got a new large message Mo"D:Trash:740
09-08 18:28:53.866 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_RECEIVED account=Mo"D folder=Trash message uid=740
09-08 18:28:53.872 V/k9      (25507): conn111226805>>> 68 UID FETCH 753 (UID BODY.PEEK[1])
09-08 18:28:54.232 V/k9      (25507): conn111226805<<<#null# [278, FETCH, [UID, 753, BODY, [1], com.fsck.k9.mail.internet.BinaryTempFileBody@d31e46a]]
09-08 18:28:54.452 V/k9      (25507): conn111226805<<<#68# [OK, FETCH completed.]
09-08 18:28:54.453 V/k9      (25507): conn111226805>>> 69 UID FETCH 753 (UID BODY.PEEK[2])
09-08 18:28:54.857 V/k9      (25507): conn111226805<<<#null# [278, FETCH, [UID, 753, BODY, [2], com.fsck.k9.mail.internet.BinaryTempFileBody@5d2bd5b]]
09-08 18:28:55.061 V/k9      (25507): conn111226805<<<#69# [OK, FETCH completed.]
09-08 18:28:55.103 D/k9      (25507): deleting temp file
09-08 18:28:55.107 D/k9      (25507): deleting temp file
09-08 18:28:55.127 V/k9      (25507): LockableDatabase: Transaction ended, took 13ms / com.fsck.k9.mailstore.LocalFolder.appendMessages(LocalFolder.java:1255)
09-08 18:28:55.138 V/k9      (25507): LockableDatabase: Transaction ended, took 6ms / com.fsck.k9.mailstore.LocalMessage.setFlag(LocalMessage.java:261)
09-08 18:28:55.141 V/k9      (25507): About to notify listeners that we got a new large message Mo"D:Trash:753
09-08 18:28:55.148 V/k9      (25507): conn111226805>>> 70 UID FETCH 754 (UID BODY.PEEK[2])
09-08 18:28:55.529 V/k9      (25507): conn111226805<<<#null# [279, FETCH, [UID, 754, BODY, [2], com.fsck.k9.mail.internet.BinaryTempFileBody@d81dd36]]
09-08 18:28:55.736 V/k9      (25507): conn111226805<<<#70# [OK, FETCH completed.]
09-08 18:28:55.737 V/k9      (25507): conn111226805>>> 71 UID FETCH 754 (UID BODY.PEEK[1])
09-08 18:28:56.101 V/k9      (25507): conn111226805<<<#null# [279, FETCH, [UID, 754, BODY, [1], com.fsck.k9.mail.internet.BinaryTempFileBody@f75d437]]
09-08 18:28:56.292 V/k9      (25507): conn111226805<<<#71# [OK, FETCH completed.]
09-08 18:28:56.343 D/k9      (25507): deleting temp file
09-08 18:28:56.346 D/k9      (25507): deleting temp file
09-08 18:28:56.367 V/k9      (25507): LockableDatabase: Transaction ended, took 14ms / com.fsck.k9.mailstore.LocalFolder.appendMessages(LocalFolder.java:1255)
09-08 18:28:56.377 V/k9      (25507): LockableDatabase: Transaction ended, took 6ms / com.fsck.k9.mailstore.LocalMessage.setFlag(LocalMessage.java:261)
09-08 18:28:56.379 V/k9      (25507): About to notify listeners that we got a new large message Mo"D:Trash:754
09-08 18:28:56.384 V/k9      (25507): conn111226805>>> 72 UID FETCH 755 (UID BODY.PEEK[1])
09-08 18:28:56.755 V/k9      (25507): conn111226805<<<#null# [280, FETCH, [UID, 755, BODY, [1], com.fsck.k9.mail.internet.BinaryTempFileBody@c0696c2]]
09-08 18:28:56.976 V/k9      (25507): conn111226805<<<#72# [OK, FETCH completed.]
09-08 18:28:56.977 V/k9      (25507): conn111226805>>> 73 UID FETCH 755 (UID BODY.PEEK[2])
09-08 18:28:57.347 V/k9      (25507): conn111226805<<<#null# [280, FETCH, [UID, 755, BODY, [2], com.fsck.k9.mail.internet.BinaryTempFileBody@cc97cd3]]
09-08 18:28:57.544 V/k9      (25507): conn111226805<<<#73# [OK, FETCH completed.]
09-08 18:28:57.586 D/k9      (25507): deleting temp file
09-08 18:28:57.590 D/k9      (25507): deleting temp file
09-08 18:28:57.609 V/k9      (25507): LockableDatabase: Transaction ended, took 13ms / com.fsck.k9.mailstore.LocalFolder.appendMessages(LocalFolder.java:1255)
09-08 18:28:57.622 V/k9      (25507): LockableDatabase: Transaction ended, took 9ms / com.fsck.k9.mailstore.LocalMessage.setFlag(LocalMessage.java:261)
09-08 18:28:57.623 V/k9      (25507): About to notify listeners that we got a new large message Mo"D:Trash:755
09-08 18:28:57.627 V/k9      (25507): conn111226805>>> 74 UID FETCH 756 (UID BODY.PEEK[2])
09-08 18:28:58.011 V/k9      (25507): conn111226805<<<#null# [281, FETCH, [UID, 756, BODY, [2], com.fsck.k9.mail.internet.BinaryTempFileBody@c2b5d0e]]
09-08 18:28:58.182 V/k9      (25507): conn111226805<<<#74# [OK, FETCH completed.]
09-08 18:28:58.183 V/k9      (25507): conn111226805>>> 75 UID FETCH 756 (UID BODY.PEEK[1])
09-08 18:28:58.547 V/k9      (25507): conn111226805<<<#null# [281, FETCH, [UID, 756, BODY, [1], com.fsck.k9.mail.internet.BinaryTempFileBody@ba5132f]]
09-08 18:28:58.775 V/k9      (25507): conn111226805<<<#75# [OK, FETCH completed.]
09-08 18:28:58.820 D/k9      (25507): deleting temp file
09-08 18:28:58.823 D/k9      (25507): deleting temp file
09-08 18:28:58.844 V/k9      (25507): LockableDatabase: Transaction ended, took 14ms / com.fsck.k9.mailstore.LocalFolder.appendMessages(LocalFolder.java:1255)
09-08 18:28:58.855 V/k9      (25507): LockableDatabase: Transaction ended, took 6ms / com.fsck.k9.mailstore.LocalMessage.setFlag(LocalMessage.java:261)
09-08 18:28:58.857 V/k9      (25507): About to notify listeners that we got a new large message Mo"D:Trash:756
09-08 18:28:58.862 V/k9      (25507): conn111226805>>> 76 UID FETCH 760 (UID BODY.PEEK[2])
09-08 18:28:59.228 V/k9      (25507): conn111226805<<<#null# [285, FETCH, [UID, 760, BODY, [2], com.fsck.k9.mail.internet.BinaryTempFileBody@ac43c1a]]
09-08 18:28:59.452 V/k9      (25507): conn111226805<<<#76# [OK, FETCH completed.]
09-08 18:28:59.454 V/k9      (25507): conn111226805>>> 77 UID FETCH 760 (UID BODY.PEEK[1])
09-08 18:28:59.821 V/k9      (25507): conn111226805<<<#null# [285, FETCH, [UID, 760, BODY, [1], com.fsck.k9.mail.internet.BinaryTempFileBody@5f9b34b]]
09-08 18:29:00.010 V/k9      (25507): conn111226805<<<#77# [OK, FETCH completed.]
09-08 18:29:00.049 D/k9      (25507): deleting temp file
09-08 18:29:00.052 D/k9      (25507): deleting temp file
09-08 18:29:00.079 V/k9      (25507): LockableDatabase: Transaction ended, took 21ms / com.fsck.k9.mailstore.LocalFolder.appendMessages(LocalFolder.java:1255)
09-08 18:29:00.089 V/k9      (25507): LockableDatabase: Transaction ended, took 6ms / com.fsck.k9.mailstore.LocalMessage.setFlag(LocalMessage.java:261)
09-08 18:29:00.090 V/k9      (25507): About to notify listeners that we got a new large message Mo"D:Trash:760
09-08 18:29:00.097 D/k9      (25507): SYNC: Done fetching large messages for folder Trash
09-08 18:29:00.097 D/k9      (25507): SYNC: About to sync flags for 12 remote messages for folder Trash
09-08 18:29:00.098 V/k9      (25507): conn111226805>>> 78 UID FETCH 764,763,762,761,759,758,757,752,751,750,749,748 (UID FLAGS)
09-08 18:29:00.456 V/k9      (25507): conn111226805<<<#null# [273, FETCH, [UID, 748, FLAGS, []]]
09-08 18:29:00.457 V/k9      (25507): Stored uid '748' for msgSeq 273 into map
09-08 18:29:00.700 V/k9      (25507): conn111226805<<<#null# [274, FETCH, [UID, 749, FLAGS, []]]
09-08 18:29:00.701 V/k9      (25507): Stored uid '749' for msgSeq 274 into map
09-08 18:29:00.701 V/k9      (25507): conn111226805<<<#null# [275, FETCH, [UID, 750, FLAGS, [\Seen]]]
09-08 18:29:00.701 V/k9      (25507): Stored uid '750' for msgSeq 275 into map
09-08 18:29:00.701 V/k9      (25507): conn111226805<<<#null# [276, FETCH, [UID, 751, FLAGS, [\Seen]]]
09-08 18:29:00.702 V/k9      (25507): Stored uid '751' for msgSeq 276 into map
09-08 18:29:00.702 V/k9      (25507): conn111226805<<<#null# [277, FETCH, [UID, 752, FLAGS, []]]
09-08 18:29:00.702 V/k9      (25507): Stored uid '752' for msgSeq 277 into map
09-08 18:29:00.702 V/k9      (25507): conn111226805<<<#null# [282, FETCH, [UID, 757, FLAGS, [\Seen]]]
09-08 18:29:00.702 V/k9      (25507): Stored uid '757' for msgSeq 282 into map
09-08 18:29:00.702 V/k9      (25507): conn111226805<<<#null# [283, FETCH, [UID, 758, FLAGS, [\Seen]]]
09-08 18:29:00.703 V/k9      (25507): Stored uid '758' for msgSeq 283 into map
09-08 18:29:00.703 V/k9      (25507): conn111226805<<<#null# [284, FETCH, [UID, 759, FLAGS, [\Seen]]]
09-08 18:29:00.703 V/k9      (25507): Stored uid '759' for msgSeq 284 into map
09-08 18:29:00.703 V/k9      (25507): conn111226805<<<#null# [286, FETCH, [UID, 761, FLAGS, [\Seen]]]
09-08 18:29:00.703 V/k9      (25507): Stored uid '761' for msgSeq 286 into map
09-08 18:29:00.704 V/k9      (25507): conn111226805<<<#null# [287, FETCH, [UID, 762, FLAGS, [\Seen]]]
09-08 18:29:00.704 V/k9      (25507): Stored uid '762' for msgSeq 287 into map
09-08 18:29:00.704 V/k9      (25507): conn111226805<<<#null# [288, FETCH, [UID, 763, FLAGS, [\Seen]]]
09-08 18:29:00.704 V/k9      (25507): Stored uid '763' for msgSeq 288 into map
09-08 18:29:00.704 V/k9      (25507): conn111226805<<<#null# [289, FETCH, [UID, 764, FLAGS, []]]
09-08 18:29:00.704 V/k9      (25507): Stored uid '764' for msgSeq 289 into map
09-08 18:29:00.705 V/k9      (25507): conn111226805<<<#78# [OK, FETCH completed.]
09-08 18:29:00.744 D/k9      (25507): SYNC: Synced remote messages for folder Trash, 8 new messages
09-08 18:29:00.771 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=K9LOCAL:bd2fbdbe-9bd6-4d25-9878-99e44025129d
09-08 18:29:00.793 V/k9      (25507): LockableDatabase: Transaction ended, took 15ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:00.796 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=758
09-08 18:29:00.817 V/k9      (25507): LockableDatabase: Transaction ended, took 13ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:00.820 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=K9LOCAL:d86f0291-ee28-414e-9140-2dbf2f0a38a1
09-08 18:29:00.838 V/k9      (25507): LockableDatabase: Transaction ended, took 10ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:00.841 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=757
09-08 18:29:00.854 V/k9      (25507): LockableDatabase: Transaction ended, took 7ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:00.855 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=760
09-08 18:29:00.871 V/k9      (25507): LockableDatabase: Transaction ended, took 10ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:00.875 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=756
09-08 18:29:00.891 V/k9      (25507): LockableDatabase: Transaction ended, took 9ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:00.895 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=755
09-08 18:29:00.915 V/k9      (25507): LockableDatabase: Transaction ended, took 13ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:00.919 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=754
09-08 18:29:00.938 V/k9      (25507): LockableDatabase: Transaction ended, took 8ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:00.948 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=753
09-08 18:29:00.965 V/k9      (25507): LockableDatabase: Transaction ended, took 10ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:00.969 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=747
09-08 18:29:00.988 V/k9      (25507): LockableDatabase: Transaction ended, took 8ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:00.994 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=740
09-08 18:29:01.025 V/k9      (25507): LockableDatabase: Transaction ended, took 12ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:01.031 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=739
09-08 18:29:01.061 V/k9      (25507): LockableDatabase: Transaction ended, took 13ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:01.066 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=738
09-08 18:29:01.084 V/k9      (25507): LockableDatabase: Transaction ended, took 11ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:01.097 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=737
09-08 18:29:01.116 V/k9      (25507): LockableDatabase: Transaction ended, took 9ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:01.120 D/k9      (25507): Broadcasted: action=com.fsck.k9.intent.action.EMAIL_DELETED account=Mo"D folder=Trash message uid=736
09-08 18:29:01.137 V/k9      (25507): LockableDatabase: Transaction ended, took 8ms / com.fsck.k9.mailstore.LocalMessage.destroy(LocalMessage.java:378)
09-08 18:29:01.160 D/k9      (25507): Done synchronizing folder Mo"D:Trash @ Fri Sep 08 18:29:01 GMT+03:00 2017 with 8 new messages
09-08 18:29:01.160 I/k9      (25507): Done synchronizing folder Mo"D:Trash
09-08 18:29:01.160 I/k9      (25507):  Command 'synchronizeMailbox' completed
09-08 18:29:01.160 I/k9      (25507): Running command 'sendPendingMessages', seq = 22(backgroundpriority)
09-08 18:29:01.162 I/k9      (25507):  Command 'sendPendingMessages' completed
```

Also, somehow it also affects sent mail, which end up with two copies in sent folder and trash folder

**Comment by droidonfire** [comment]
I can confirm the same issue, but only while using k9 with imap.

**Comment by phmarek** [comment]
I even get duplicated mails in the INBOX (via imap); reading one of the duplicated mails marks the other one "read" as well, so it looks as if it's only displayed twice and not really a duplicated record?!

Didn't happen with the previous version of k9mail.

**Comment by phmarek** [comment]
Confirmed via "adb shell".
```
# cd /data/data/com.fsck.k9/databases
# sqlite3 *.db
sqlite> select id,uid,date,message_id,deleted from messages where folder_id = 1 and deleted=0 order by date desc limit 10;
```

No sign of duplications, but emails are listed twice in INBOX.

**Comment by rancidfrog** [comment]
Is more information needed here?
It seems to be a synchronization issue between local folder and server.
However, in server there is also duplicate message, (sent folder, probably deleted too).

**Comment by Avispa** [comment]
A year later I can still confirm this issue. I have a doubled trash as well as doubled outbox.

I can add that I have multiple accounts even at the same mailbox provider and the issue only appears in one account (the first one and all IMAP, in case it makes and difference).

**Comment by faginbagin** [comment]
I also see this problem on both oldish (tablet w/ android 6.0.1) and newish (phone w/ android 7.1.1) devices, both running k-9 mail 5.600. Like Avispa, I have two accounts, both IMAP. The first account is connected to a Linux debian dovecot IMAP server. The second is a gmail account. The duplicate Trash is seen on the linux account. The gmail account is very quiet, so I haven't noticed the problem there. Steps to reproduce:
1) Delete email from Inbox.
2) Switch to Trash, the deleted email is shown.
3) Click on Refresh (necessary to empty Trash). Duplicate of deleted email is shown.

If I refresh Trash on other device with K-( Mail, it doesn't show duplicates, only on the device where email was deleted. Since I can still empty Trash, the problem is annoying but not annoying enough to install adb to capture a log.

**Comment by C-Cur** [comment]
Is there a solution to this problem.
I am running K-9 Mail version 5.600 freshly updated on an android smartphone, (Samsung Galaxy S5), and using it in IMAP on a GMX mail account.
Deleted emails appear twice in the trash for me too.
If there is a solution, can someone please post it.
TIA

**Comment by Adambean** [comment]
This seems to happen for me too when moving a message but only on IMAP accounts served by hMailServer. The duplicate messages are not visible when looking at the mailbox via another client such as Thunderbird.

Using the "clear messages (danger!)" function  resolves this. When the mailbox syncs again the duplicates are gone. Next time you move a message it will duplicate as before though.

**Comment by gdt** [comment]
I recently saw behavior that may be related.   I am using IMAP, with Trash as an IMAP folder, and delete by moving to Trash, to align with gnus. I I don't keep Trash synchronized. Mostly everything works fine.   i had deleted a message and realized I wanted it.  One can't move an unsynchronized message from Trash back to INBOX (a separate issue).

So I hit the sync button, which normally puts the local deleted messages  into IMAP and gets the ones deleted by other clients.  Usually this works without problems, and I can then refile.  But this time, I entered the message right away and tried to refile after I thought the sync must have finished, but it still refused.  I went back to folder view and there were two copies. The other copy could be moved, and the first copy went away on another sync.  I have not tried to reproduce this.

So my theory, without any real basis, is that when a local message is synced back to IMAP, that causes a new message object representing the IMAP version and the original local one should be and normally is deleted, unless for some reason it isn't.   I wildly speculate that the reporter's issue is related to this, which of course might show up in debug log.

**Comment by Schwefelholz** [comment]
I'm seeing the same issue here on at least two devices: OnePlus 5 (Android 9) and Samsung Galaxy Tab S6 (Android 10). Both devices run K9 V 5.600.

I'm observing that if a mail gets deleted in K9, a PC using Thunderbird just lists one deleted mail (while K9 lists two copies) and once this mail is removed from trash in Thunderbird, one(!) copy is removed in K9 as well.
